ipaplatform/base/services.py

```diff
@@ -93,9 +92,17 @@ class PlatformService(object):
 
     """
 
-    def __init__(self, service_name, api=ipalib.api):
+    def __init__(self, service_name, api=None):
         self.service_name = service_name
-        self.api = api
+        if api is not None:
+            self.api = api
+        else:
+            import ipalib  # FixMe: break import cycle
+            self.api = ipalib.api
```

Comment:
Maybe would be better to make this comment to code, I'm not sure if we should have `else` branch there. We never declared this as a public stable API.